README.md

```diff
@@ -108,9 +107,11 @@ On Windows, start a new shell with administrative privileges, for
 example by <kbd>Windows Key</kbd>, then type `cmd.exe`, right click on its menu
 entry, and choose `Run as administrator`. In the new administrative
 shell, cd to the path, where you have compiled v.exe, then type:
-```bat
+
+```bash
 .\v.exe symlink
```

Comment:
This should not be changed, [bat is a valid language according to the GitHub Linguist](https://github.com/github/linguist/blob/master/lib/linguist/languages.yml#L416).